lib/db.js

```diff
@@ -1,7 +1,8 @@
-
-let Promise = require('bluebird');
 const logger = require('./logger').db;
 let config = require('./config');
+const redisCommands = require('redis-commands');
```

Comment:
Instead of adding a new dependency, can we just add this as a constant?

I ran this command:

```shell
curl -s https://raw.githubusercontent.com/NodeRedis/redis-commands/master/commands.json | json -M -a key | awk -v ORS=", " -v q="\"" '{ print q $1 q }' | sed 's/, $//'| awk '{ print "[",$0,"]" }' | json
```
It generates a JSON array that we can then shove into a JavaScript module.

```js
const REDIS_COMMANDS = [
  "append",
  "asking",
  "auth",
  "bgrewriteaof",
  "bgsave",
  "bitcount",
  "bitfield",
  "bitop",
  "bitpos",
  "blpop",
  "brpop",
  "brpoplpush",
  "client",
  "cluster",
  "command",
  "config",
  "dbsize",
  "debug",
  "decr",
  "decrby",
  "del",
  "discard",
  "dump",
  "echo",
  "eval",
  "evalsha",
  "exec",
  "exists",
  "expire",
  "expireat",
  "flushall",
  "flushdb",
  "geoadd",
  "geodist",
  "geohash",
  "geopos",
  "georadius",
  "georadiusbymember",
  "get",
  "getbit",
  "getrange",
  "getset",
  "hdel",
  "hexists",
  "hget",
  "hgetall",
  "hincrby",
  "hincrbyfloat",
  "hkeys",
  "hlen",
  "hmget",
  "hmset",
  "host:",
  "hscan",
  "hset",
  "hsetnx",
  "hstrlen",
  "hvals",
  "incr",
  "incrby",
  "incrbyfloat",
  "info",
  "keys",
  "lastsave",
  "latency",
  "lindex",
  "linsert",
  "llen",
  "lpop",
  "lpush",
  "lpushx",
  "lrange",
  "lrem",
  "lset",
  "ltrim",
  "memory",
  "mget",
  "migrate",
  "module",
  "monitor",
  "move",
  "mset",
  "msetnx",
  "multi",
  "object",
  "persist",
  "pexpire",
  "pexpireat",
  "pfadd",
  "pfcount",
  "pfdebug",
  "pfmerge",
  "pfselftest",
  "ping",
  "post",
  "psetex",
  "psubscribe",
  "psync",
  "pttl",
  "publish",
  "pubsub",
  "punsubscribe",
  "quit",
  "randomkey",
  "readonly",
  "readwrite",
  "rename",
  "renamenx",
  "replconf",
  "restore",
  "restore-asking",
  "role",
  "rpop",
  "rpoplpush",
  "rpush",
  "rpushx",
  "sadd",
  "save",
  "scan",
  "scard",
  "script",
  "sdiff",
  "sdiffstore",
  "select",
  "set",
  "setbit",
  "setex",
  "setnx",
  "setrange",
  "shutdown",
  "sinter",
  "sinterstore",
  "sismember",
  "slaveof",
  "slowlog",
  "smembers",
  "smove",
  "sort",
  "spop",
  "srandmember",
  "srem",
  "sscan",
  "strlen",
  "subscribe",
  "substr",
  "sunion",
  "sunionstore",
  "swapdb",
  "sync",
  "time",
  "touch",
  "ttl",
  "type",
  "unlink",
  "unsubscribe",
  "unwatch",
  "wait",
  "watch",
  "zadd",
  "zcard",
  "zcount",
  "zincrby",
  "zinterstore",
  "zlexcount",
  "zrange",
  "zrangebylex",
  "zrangebyscore",
  "zrank",
  "zrem",
  "zremrangebylex",
  "zremrangebyrank",
  "zremrangebyscore",
  "zrevrange",
  "zrevrangebylex",
  "zrevrangebyscore",
  "zrevrank",
  "zscan",
  "zscore",
  "zunionstore"
];

Object.freeze(REDIS_COMMANDS)

module.exports = REDIS_COMMANDS;
```